DataStructureFour/JobTest.java

```diff
@@ -0,0 +1,53 @@
+package DataStructureFour;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import DataStructureFour.beans.DepartmentChair;
+import DataStructureFour.beans.Graduates;
+import DataStructureFour.beans.Professors;
+import DataStructureFour.beans.UnderGraduates;
+import DataStructureFour.service.Printer;
+
+public class JobTest {
+
+	DepartmentChair departmentChair;
+	Professors professors;
+	Graduates graduates;
+	UnderGraduates underGraduates;
+	List<String> actualList;
+	List<String> expectedList;
+	@Before
+	public void setUp(){
+		departmentChair = new DepartmentChair("Print Department chair");
```

Comment:
I think you can directly use instance of Entity class. No need to extend this class by seperate child classes. You can have one more property like 'Entity Type' in entity class and the you do not need to create seperate classes for each entity type. For example for departmentChair  pass these in constructor :  new DepartmentChair("Print Department chair", "departmentChair"); and in constructor based on the type passed you can set priority. In this way you do not need to extend Entity class for each child type class.